### ISSUE TITLE
Odd export BooKMark behaviour

### ISSUE THREAD

**Issue by GitHubRulesOK** [sumatrapdfreader/sumatrapdf#1628]
Whilst testing forum issue https://forum.sumatrapdfreader.org/t/open-pdf-to-a-specific-bookmarked-page/3090/5

[forum BkMkTest.pdf](https://github.com/sumatrapdfreader/sumatrapdf/files/4806941/forum.BkMkTest.pdf)

I noted several issues
The request was for SumatraPDF to be enhanced to automatically open on Page 2 (An aleady open issue as Part 4b of .bkm changes #1407)

HOWEVER this issue is
The bookmarks work to a degree but when queried via export bookmarks the output is contantly varying e.g two examples from the same file are

```
version: 998
name: default view
"Page 1" page:1 destkind:scrollTo destname:#1 pos:997985,997985
"Page 2" page:2 destkind:scrollTo destname:#2 pos:998033,998033
"Page 3" page:3 destkind:scrollTo destname:#3 pos:998153,998153
```
and
```
version: 998
name: default view
"Page 1" page:1 destkind:scrollTo destname:#1 pos:-1.71166e-15,-1.71166e-15
"Page 2" page:2 destkind:scrollTo destname:#2 pos:-1.71203e-15,-1.71203e-15
"Page 3" page:3 destkind:scrollTo destname:#3 pos:-1.71187e-15,-1.71187e-15
```

**Comment by GitHubRulesOK** [comment]
@kjk 
looks like it occured as a result of don't crash if dti is is null (fixes #1611)

**Comment by GitHubRulesOK** [comment]
Seems this was resolved some time ago
